tests/sentry/api/endpoints/test_project_template_detail.py

```diff
@@ -64,6 +64,81 @@ def test_get__no_access(self):
         assert response.status_code == 403
 
 
+class ProjectTemplateUpdateTest(ProjectTemplateAPIBase):
+    endpoint = "sentry-api-0-organization-project-template-detail"
+    method = "put"
+
+    def test_put__no_feature(self):
+        response = self.get_error_response(
+            self.organization.id, self.project_template.id, status_code=404
+        )
+        assert response.status_code == 404
+
+    @with_feature(PROJECT_TEMPLATE_FEATURE_FLAG)
+    def test_put__only_name(self):
```

Comment:
The name here indicates that it's only updating the name, but it passes both the name and the options

Reply:
ty - nice catch.